plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -81,7 +81,10 @@ class NewCreateServiceModalForm extends Component {
     );
 
     // Render initial app config
-    this.state.appConfig = this.getAppConfig(this.state);
+    this.state.appConfig = this.getAppConfig(
```

Comment:
This looks a bit messy. Let's create local variables and then set state after these calculations.